website/content/en/releases/13.1R/relnotes.adoc

```diff
@@ -37,7 +37,12 @@ All users are encouraged to consult the release errata before installing FreeBSD
 
 This document describes the most user-visible new or changed features in FreeBSD since {releasePrev}. In general, changes described here are unique to the {releaseBranch} branch unless specifically marked as MERGED features.
 
-Typical release note items document recent security advisories issued after {releasePrev}, new drivers or hardware support, new commands or options, major bug fixes, or contributed software upgrades. They may also list changes to major ports/packages or release engineering practices. Clearly the release notes cannot list every single change made to FreeBSD between releases; this document focuses primarily on security advisories, user-visible changes, and major architectural improvements.
+Typical release notes: 
+
+* include recent security advisories issued after {releasePrev}, new drivers or hardware support, new commands or options, major bug fixes, or contributed software upgrades
+* may also include changes to major ports/packages or release engineering practices. 
+
+Release note documents such as this do not list every change between releases. This document focuses primarily on security advisories, user-visible changes, and major architectural improvements.
```

Comment:
This is the fourth mention of security advisories. 

Deduplication of things such as this might yield a much more concise abstract and introduction.